src/stepDefinitions/sharingContext.js

```diff
@@ -286,10 +286,16 @@ Given(
 )
 
 Given('the administrator has enabled exclude groups from sharing', function() {
+    if (client.globals.ocis) {
+        return
+    }
   return runOcc(['config:app:set core shareapi_exclude_groups --value=yes'])
 })
 
 Given('the administrator has excluded group {string} from sharing', async function(group) {
+    if (client.globals.ocis) {
+        return
+    }
```

Comment:
@amrita-shrestha alternatively if we put this here in this function we can remove all the extra added if conditions
https://github.com/owncloud/owncloud-test-middleware/blob/ec80dfa391e6068774ac73ae5cf0a2018cf7989b/src/helpers/occHelper.js#L8